arrow-site/docs/_data/doc-versions.yml

```diff
@@ -22,13 +22,17 @@ docVersions:
 
   - title: "v1.0.1"
     url: https://arrow-kt.io/
-    previous: https://arrow-kt.io/docs/1.0.0/
+    previous: https://arrow-kt.io/docs/1.0.1/
 
   - title: "v1.1.0"
     url: https://arrow-kt.io/
-    previous: https://arrow-kt.io/docs/1.0.1/
+    previous: https://arrow-kt.io/docs/1.1.0/
+
+  - title: "v1.1.1"
+    url: https://arrow-kt.io/
+    previous: https://arrow-kt.io/docs/1.1.1/
```

Comment:
For the above changes on this file, I believe the previous for:
- `1.0.1` should be `1.0.0`
- `1.1.0` should be `1.0.1`
- `1.1.1` should be `1.1.0`

It would make sense for all of those versions to be considered to have the same docs, but even then their `previous` should all point to `0.13`.

On a tangent note, most of these versioned links don't work right now (maybe they'll be published and corrected with this release, in that case please ignore this point):
```

- https://arrow-kt.io/docs/0.9/ doesn't work
- https://arrow-kt.io/docs/0.10/ doesn't work
- https://arrow-kt.io/docs/0.11/ doesn't work
- https://arrow-kt.io/docs/0.12/ doesn't work
  - but https://arrow-kt.io/docs/0.12/core/ works
- https://arrow-kt.io/docs/0.13/ doesn't work
  - but https://arrow-kt.io/docs/0.13/core/ works
- https://arrow-kt.io/docs/1.0.1/ doesn't work
- https://arrow-kt.io/docs/1.1.0/ and https://arrow-kt.io/docs/next/ work
  - but then, the `/core/` page under these has a `PREVIOUS` link to `/docs/$v/core/core/`, which is a 404
```